src/config.h.in

```diff
@@ -0,0 +1 @@
+#cmakedefine MIXXX_INSTALL_DOCDIR_RELATIVE_TO_DATADIR "@MIXXX_INSTALL_DOCDIR_RELATIVE_TO_DATADIR@"
```

Comment:
By the way, we should check if we can move more declarations to this file. For global defines such as `__LINUX__` this probably doesn't make much sense because it would be hard to debug if someone forgets to I clude `config.h` but for stuff that isn't needed everywhere this could bring down recompilation times when these values change, because you only need to recompile files that actually use it.